### DISCUSSION TITLE
Add custom button in "Brushes" for transparent background

### DISCUSSION THREAD

**Discussion by roblo783** [xdan/jodit#699] [Q&A] [answered]
![Custom_button_brushes](https://user-images.githubusercontent.com/63327684/123994513-bac61980-d9cd-11eb-9508-228f5b2d8236.PNG)

I want to add a custom button in "Brushes", next to the color picker, see screenshot. When a user clicks on that button, and the user has selected (marked) a text, the text background should be set to "transparent".

I have seen example of adding a custom button, but not in "brushes". How do I do that?

**Comment by xdan** [comment, marked as answer]
Putting a button *inside* the brush dropdown means overriding the color-picker control, which is fiddly — but if you just want a one-click "transparent background", a separate toolbar button is easy. It runs the same command the brush uses:

```js
editor.execCommand('background', false, 'transparent');
```

Register it through `controls` / `extraButtons` with that `exec`, and it'll apply to the current selection. Want the full button snippet?
